galgebra/metric.py

```diff
@@ -125,6 +125,82 @@ def square_root_of_expr(expr):
 
 
 def symbols_list(s, indices=None, sub=True, commutative=False):
+    """
+    Convert a string to a list of symbols.
+
+    If :class:`galgebra.printer.Eprint` is enabled, the symbol names will
+    contain ANSI escape sequences.
+
+    Parameters
+    ----------
+    s : str
+        Specification. If `indices` is specified, then this is just a prefix.
+        If `indices` is not specified then this is a string of one of the forms:
+
+        * ``prefix + "*" + index_1 + "|" + index_2 + "|" + ... + index_n``
+        * ``prefix + "*" + n_indices``
+        * ``name_1 + "," + name_2 + "," + ... + name_n``
+        * ``name_1 + " " + name_2 + " " + ... + name_n``
+
+    indices : list, optional
+        List of indices to append to the prefix.
+    sub : bool
+        If true, mark as subscript separating prefix and suffix with `_`, else
+        mark as superscript using `__`.
+    commutative : bool
+        Passed on to :class:`sympy.Symbol`.
+
+    Returns
+    -------
+    symbols : list of :class:`sympy.Symbol`
+
+    Examples
+    --------
+
+    Names can be comma or space separated:
+
+    >>> symbols_list('a,b,c')
+    [a, b, c]
+    >>> symbols_list('a b c')
+    [a, b, c]
+
+    Mixing commas and spaces gives surprising results:
+
+    >>> symbols_list('a b,c')
+    [a b, c]
```

Comment:
😲 
Should this be considered as a bug? Either we should handle it properly or reject it, I prefer the latter.